CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 For more information on the list of deprecated flags and APIs, have a look at
 https://docs.docker.com/engine/deprecated/ where you can find the target removal dates 
 
+## 18.06.3-ce (2019-02-19)
+
```

Comment:
Do we want to mention the packaging-fix in this release?

Reply:
yes, sorry this is PR is WIP. i'll add words.